### PR TITLE
Makedag bug fix

### PR DESCRIPTION
**Pull request by benbaley**

### CODE REVIEW

File: core/parallel_context.go [hunk 1]
```diff
@@ -196,12 +198,19 @@ func (ctx *ParallelContext) buildTransferSuccessResult(idx int, fromStateObject,
 		err:             nil,
 	}
 	ctx.SetResult(idx, result)
-	/*	log.Debug("Execute trasnfer success", "blockNumber", ctx.header.Number.Uint64(), "txIdx", idx, "txHash", tx.Hash().TerminalString(),
+	/*receiptString := fmt.Sprintf("cumulative: %v gas: %v contract: %v status: %v tx: %v logs: %v bloom: %x state: %x\n",
```
Comment (mowind) [inline core/parallel_context.go]: suggest delete unused code

File: core/parallel_executor.go [hunk 2]
```diff
@@ -1,15 +1,15 @@
 package core
 
 import (
+	"github.com/AlayaNetwork/Alaya-Go/crypto"
 	"math/big"
```
Comment (mowind) [inline core/parallel_executor.go:5]: fmt

File: core/parallel_executor.go [hunk 3]
```diff
@@ -143,14 +143,13 @@ func (exe *Executor) ExecuteTransactions(ctx *ParallelContext) error {
...
 
-	/*
-		// dag print info
-		logVerbosity := debug.GetLogVerbosity()
-		if logVerbosity == log.LvlTrace {
-			inf := ctx.txListInfo()
-			log.Trace("TxList Info", "blockNumber", ctx.header.Number, "txList", inf)
-		}
-	*/
+	// dag print info
```
Comment (mowind) [inline core/parallel_executor.go:142]: suggest delete unused code

File: core/parallel_executor.go [hunk 4]
```diff
@@ -232,21 +231,31 @@ func (exe *Executor) executeContractTransaction(ctx *ParallelContext, idx int) {
 	ctx.AddPackedTx(tx)
 	ctx.GetState().IncreaseTxIdx()
 	ctx.AddReceipt(receipt)
+	//receiptString := fmt.Sprintf("cumulative: %v gas: %v contract: %v status: %v tx: %v logs: %v bloom: %x state: %x\n",
```
Comment (mowind) [inline core/parallel_executor.go]: suggest delete unused code

File: core/parallel_state_processor.go [hunk 5]
```diff
@@ -1,6 +1,7 @@
 package core
 
 import (
+	"github.com/AlayaNetwork/Alaya-Go/common"
 	"time"
```
Comment (mowind) [inline core/parallel_state_processor.go:5]: fmt

File: core/parallel_context.go (hunk 1 above)
Comment (benbaley) [reply core/parallel_context.go]: done

File: core/types.go [hunk 6]
```diff
@@ -17,6 +17,7 @@
 package core
 
 import (
+	"github.com/AlayaNetwork/Alaya-Go/common"
```
Comment (mowind) [inline core/types.go:20]: fmt

File: core/types/block.go [hunk 7]
```diff
@@ -313,6 +313,16 @@ func NewBlock(header *Header, txs []*Transaction, receipts []*Receipt) *Block {
 		copy(b.transactions, txs)
 	}
 
+	/*if header != nil && header.Number != nil && receipts != nil {
```
Comment (mowind) [inline core/types/block.go]: suggest delete unused code

**Pull request merged**